MWVideoGridPlugin.podspec

```diff
@@ -16,6 +16,8 @@ Pod::Spec.new do |s|
 	
     s.subspec 'Core' do |cs|
 	    cs.dependency            'MobileWorkflow'
+        cs.dependency            'Kingfisher', '~> 6.0'
+        cs.dependency            'FancyScrollView', '0.1.3'
```

Comment:
@FWJonathan `FancyScrollView` is the dependency that [we forked](https://github.com/FutureWorkshops/FancyScrollView) to our own GitHub account and [published it](https://github.com/FutureWorkshops/MWPodspecs/tree/main/FancyScrollView/0.1.3) on `MWPodspecs`. Will this just work or should I be pointing to it somehow instead of just using the name?

Reply:
Hmm, I think you might need to rename the forked pod or it will use the public one.

Reply:
Maybe updating the pod spec name should be enough, e.g. MWFancyScrollView.podspec (with corresponding name within the podspec).

Reply:
@FWJonathan There's no public one (the fork was to add support for Cocoapods since the original library did not). It seems that you can't specify a `:git =>` on the `.dependency`: https://github.com/CocoaPods/CocoaPods/issues/8550

I'd assume that it'll work since `MWPodspecs` is public, it should just grab that one, right? Do you know how can I test this assumption? 🤔

Reply:
Ah yes, then I guess we should just publish the plugin pod and see if it installs correctly?